docs/source/framework.rst

```diff
@@ -12,3 +12,9 @@ Model
 
 .. autoclass:: Model
     :members:
+
+Model Configuration Error
+------------
+
+.. autoclass:: ModelConfigurationError
```

Comment:
No need to document this as it is only for an internal usage.

Reply:
remove